command/v7/service_command.go

```diff
@@ -177,15 +184,7 @@ func (cmd ServiceCommand) displayParametersEmpty() {
 }
 
 func (cmd ServiceCommand) displayParametersData() {
-	cmd.UI.DisplayTextWithFlavor(
-		"Showing parameters for service instance {{.ServiceInstanceName}}...",
-		map[string]interface{}{
-			"ServiceInstanceName": cmd.serviceInstance.Name,
-		},
-	)
-	cmd.UI.DisplayNewline()
-
-	data, err := json.Marshal(cmd.serviceInstance.Parameters.Value)
+	data, err := json.MarshalIndent(cmd.serviceInstance.Parameters.Value, "", "  ")
 	if err != nil {
 		panic(err)
```

Comment:
I think I wrote this originally. What do you think about it? The reasoning is that since the parameters were once JSON, it's impossible for the Marshal to return an error - since errors would be because of types that cannot be marshalled. However looking at it again, it looks like a code small. I wonder if there's a better way to encapsulate it? Or just return the error so that the code looks "normal" even though there will never be an error?

Reply:
Hmm - What about just ignoring the error. We are confident that it is JSON and there are tests around the result.